### PR TITLE
Prepare changelog for v0.19

### PR DESCRIPTION
**Pull request by registerrier**
**Description**

This pull request prepares the changeling for the coming release. 

**Dear reviewer**

please comment, update etc.

### CODE REVIEW

**Review by Bultako — COMMENTED** [review]
Thanks @registerrier 
I've place some comments in line.

File: CHANGES.rst [hunk 1]
```diff
@@ -2,7 +2,133 @@
...
+to freeze all spectral or spatial parameters of a given ``Models`` object, to select all
+components of a ``Models`` lying inside a given region.
+
+Documentation has been significantly reorganized. In particular, the tutorials have been
+restructured into subfolders based of the type of treatment described. Tutorials are now
+presented through a nice graphical gallery. Some tutorials have been removed because their
+objectives are too specific for the general documentation. To present these specific, possibly
+contributed examples, a new web page and repository has been created: `gammapy.recipes`
+(https://github.com/gammapy/gammapy-recipes).
+
```
[the remark this answers is not in the document]
Comment (registerrier) [reply CHANGES.rst:40]: Good idea.

**Comment by adonath** [comment]
@registerrier Please rebase...

**Review by adonath — APPROVED** [review]
Thanks @registerrier, no further comments from my side! I'll merge now and the we go ahead with the release..

**Pull request merged**